Dockerfile.build.tmpl

```diff
@@ -177,15 +177,13 @@ RUN pip3 install --upgrade native_client/ctcdecode/dist/*.whl
 ENV PYTHONIOENCODING UTF-8
 
 # Build KenLM in /DeepSpeech/native_client/kenlm folder
-WORKDIR /DeepSpeech/native_client
-RUN rm -rf kenlm && \
-	git clone https://github.com/kpu/kenlm && \
-	cd kenlm && \
-	git checkout 87e85e66c99ceff1fab2500a7c60c01da7315eec && \
-	mkdir -p build && \
-	cd build && \
-	cmake .. && \
-	make -j $(nproc)
+WORKDIR /DeepSpeech/native_client/kenlm
+ENV EIGEN3_ROOT=/DeepSpeech/native_client/kenlm/eigen-eigen-07105f7124f9
+RUN wget -O - https://gitlab.com/libeigen/eigen/-/archive/3.3.8/eigen-3.3.8.tar.bz2 | tar xj
+RUN mkdir -p build && \
+    cd build && \
+    cmake .. && \
+    make -j $(nproc)
```

Comment:
what

Reply:
what, what?

Reply:
Why do we need to update the build steps to move from CircleCI to GitHub Actions? And if it's unrelated to the move, why is not on (at least) a separate commit with an explanation?

Reply:
Oh yes, that's unrelated: since GitHub Actions, we now have a `kenlm` submodule, and when I was looking over the changes to get rid of `taskcluster` reference I took the opportunity to move to using submodule one.

I'll make a separate commit :)